Tests/MetaTests/Extensions/Snapshotting+Codegen.swift

```diff
@@ -15,4 +16,14 @@ extension Snapshotting where Value == URL, Format == String {
     }
     
     static let copyMethod: Snapshotting<URL, String> = .generatedCode(CopyVisitor())
```

Comment:
you should delete this hardcoded `generatedCode` and use instead `generatedCode<T: CodegenVisitor>(_ visitor: T)` where you need it

Reply:
It is just a utility for brevity, I think it is nice to have more readable tests, even though it relies in the parametric one.